core/src/main/java/org/testcontainers/dockerclient/LogToStringContainerCallback.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.dockerclient;
+
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.core.command.LogContainerResultCallback;
+
+public class LogToStringContainerCallback extends LogContainerResultCallback {
```

Comment:
No problem then - the type system will at least help people not use the wrong thing.